python/tk_desktop2/websockets/requests/sgc_actions/open_task.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2018 Autodesk, Inc.  All rights reserved.
+#
+# Use of this software is subject to the terms of the Autodesk license agreement
+# provided at the time of installation or download, or which otherwise accompanies
+# this software in either electronic or hard copy form.
+#
+
+import sgtk
+from ..request import WebsocketsRequest
+from ....shotgun_entity_path import ShotgunEntityPath
+
+logger = sgtk.LogManager.get_logger(__name__)
+
+
+class OpenTaskInSGCreateWebsocketsRequest(WebsocketsRequest):
```

Comment:
Should we be naming things as interacting with "Tasks" when we already support both Task and Version entities, with the likelihood of future support for Playlist and Cut/CutItem entity types? Maybe it should all be `OpenEntity...` across the board.

Reply:
good question - there used to be an `OpenTaskInSGCreateWebsocketsRequest` AND an `OpenVersionInSGCreateWebsocketsRequest`- which seemed to make sense to me (as well as being nice and ortohogonal, but i got feedback from @eidmi and the create guys that they would prefer a single method to represent both - i think I don't fully understand the conceptual model here, but i think the way SC thinks about tasks and versions is somehow very similar... a version is like a task which is displayed in a particular 'mode' sort of thing....